cookbooks/ey-cron/recipes/default.rb

```diff
@@ -70,6 +70,11 @@
   recursive true
 end
 
+file "/var/spool/cron/crontabs/#{node['owner_name']}" do
```

Comment:
Wouldn't it be better to fix it at the root cause: https://github.com/engineyard/ey-cookbooks-stable-v6/blob/26d47c289ac549ce1e15de782be709e0fab1abf6/cookbooks/ey-cron/recipes/default.rb#L31?

Reply:
That code block is not always executed, therefore it's better to handle this separately